static/app/components/acl/access.tsx

```diff
@@ -17,20 +15,23 @@ type ChildRenderProps = {
 type ChildFunction = (props: ChildRenderProps) => any;
 
 type Props = {
-  organization: Organization;
   /**
    * List of required access levels
    */
-  access?: Scope[];
+  access: Scope[];
   /**
    * Children can be a node or a function as child.
    */
-  children?: React.ReactNode | ChildFunction;
-
+  children: React.ReactNode | ChildFunction;
   /**
    * Requires superuser
    */
   isSuperuser?: boolean;
+  /**
+   * Evaluate access against a defined organization. If this is not provided,
+   * the access is evaluated against the currently active organization.
+   */
+  organization?: Organization;
```

Comment:
is this prop something to maintain?

Reply:
Yeah, I thought so too, but when I tried removing it I found a single place somewhere in settings I think where it's passing some other org :/